.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -1,7 +1,6 @@
 ---
 name: 🐛 Bug Report
 about: If something isn't working as expected.
-
```

Comment:
Sure this isn't needed for GitHub? I feel like this empty line was included for some reason.

Reply:
I don't think so, but I can test on a new repo.

Reply:
Okay, so this is how GitHub initially creates default issue templates through the UI. However, I don't see a difference here. Anyway, I'll revert this changes, since GitHub also initially adds a new line here.